src/ni/vsbuild/steps/LvVITesterStep.groovy

```diff
@@ -0,0 +1,18 @@
+package ni.vsbuild.steps
+
+import ni.vsbuild.BuildConfiguration
+
+class LvVITesterStep extends LvProjectStep {
+
+   def Test_path
+
+   LvVITesterStep(script, mapStep, lvVersion) {
+      super(script, mapStep, lvVersion)
+      this.Test_path = mapStep.get('Test_path')
```

Comment:
The variable in TOML should be `test_path` to match the existing conventions.

Reply:
Updated to `this.testPath = mapStep.get('test_path')`